### ISSUE TITLE
Merge with MacGap

### ISSUE THREAD

**Issue by ricardobeat** [pojala/electrino#10]
A long time ago, in a galaxy far far away: https://github.com/MacGapProject/MacGap1

macgap has much wider API support, but has been suffering from lack of interest. How about we direct some of this newfound hunger for thin apps into making it alive again?

**Comment by tbodt** [comment]
But MacGap isn't cross-platform...

**Comment by ricardobeat** [comment]
Neither is Electrino at the moment... I guess what I'm saying is there is already a base to build upon instead of starting again from scratch, then time can be spent on making it actually cross-platform - right now this is nothing new.

**Comment by styfle** [comment]
It seems like some of the source code would be valuable but the two projects have different goals, specifically the exposed API.

**Comment by pojala** [comment]
Yes, the difference to MacGap is that Electrino should ideally provide a strict subset of Electron APIs.

The licenses are compatible, so nothing prevents sharing code though!

**Comment by taoeffect** [comment]
This sounds like a great idea. 👍 

Just hope it can be done in a cross-platform way, as that's the entire benefit of Electron. Given that though, I assume that it will be.
